tests/sqlparser_common.rs

```diff
@@ -3120,6 +3120,103 @@ fn parse_interval() {
     );
 }
 
+#[test]
+fn parse_interval_and_or_xor() {
```

Comment:
@sarahyurick please try to add tests considering the `FROM -> TO` being equal as well, if possible. It's nice to have a known about which queries are parsed the same as the original one.

Reply:
Sure, just to clarify do you mean tests with something like `INTERVAL '1-1' YEAR TO MONTH` in them?

Reply:
Got it, thanks!